docs/src/administration/web/configure-project.md

```diff
@@ -9,10 +9,19 @@ description: |
 
 ## General
 
-From the first page of the project settings, `General`, you can update the project name, or navigate to other project settings options on the left side of the screen.
+From the first page of the project settings, `General`, you can update the project name, set the project timezone, and navigate to other project settings options on the left side of the screen.
 
 ![configure project](/images/management-console/settings-basics.png "0.7")
 
+The project timezone affects [automated backups](../../dedicated/overview/backups.md).
+Changes on the automated backup schedule might affect your own maintenance window.
```

Comment:
I don't know that this is so universally true to be worth mentioning. People might use the project timezone for such windows, but I don't think they'd go through the backup schedule. I think it'd be a query for the timezone and using that directly. That's the idea that was proposed, at least.